src/main/java/gov/cabinetoffice/gap/adminbackend/services/GrantAdvertService.java

```diff
@@ -426,15 +417,14 @@ private void createRichTextQuestionsInContentful(final GrantAdvert advert, final
                         .anyMatch(qr -> qr.getId().equals(q.getId())))
                 .toList();
 
-        if (responses != null && !responses.isEmpty()) {
+        if (!responses.isEmpty()) {
```

Comment:
Any reason the null check has been removed? not sure if the isEmpty does it (depends what implementation is used)

Reply:
Just an IDE recommendation - impossible for it to be null